wwwroot/js/site.js

```diff
@@ -10,4 +10,23 @@ $(document).ready(function () {
         location.replace("/");
     });
   });
+
+  $("#NewProduct_ProductTypeId").on("change", function () {
+
+    var formData = {
+      name: $('#NewProduct_Name').val(),
+      description: $('#NewProduct_Description').val(),
+      price: parseFloat($('#NewProduct_Price').val())   

```

Comment:
Made Price a float thinking it would help with conversion to decimal in C# but no success yet.